docs/_includes/generated/jmx_server_configuration.html

```diff
@@ -0,0 +1,18 @@
+<table class="table table-bordered">
```

Comment:
Why is this file generated now? (I don't see a code change that triggers this)

Reply:
this is actually due to the fact that doc gen only regenerate files (and if there's deletion, items will be deleted) but it doesn't remove htmls when something is entirely removed. I manually removed this for one of my fault doc gen